frontend/catalyst/pennylane_extensions.py

```diff
@@ -2446,3 +2448,52 @@ def _get_batch_size(args_flat, axes_flat, axis_size):
         )
 
     return batch_size
+
+
+def callback(func):
+    """Decorator that will correctly pass the signature as arguments to the callback
+    implementation.
+
+    At the moment, this is a stub.
+    We can also use here isTracing to only bind with the callback while it is tracing,
+    otherwise, call the function itself.
+    """
```

Comment:
@erick-xanadu don't forget to update the changelog, and also rewrite this docstring before merging!

Reply:
@josh146 I will be changing the changelog as the other PRs get merged: https://github.com/PennyLaneAI/catalyst/pull/540/commits/9cdfa7bc3ceecfdf3408fd475505fb769ad54e59 https://github.com/PennyLaneAI/catalyst/pull/540/commits/a757186ab5cc48e307a2f949a59b421b500c5200 Please take a look at the changes